build/repos/opstools.repo

```diff
@@ -1,6 +1,19 @@
+# CentOS-OpsTools.repo
```

Comment:
It's not duplicate. Build task is using a repo file instead of fetching it.

[1] https://github.com/infrawatch/sg-core/blob/master/build/Dockerfile#L10